gossip/ethapi_backend.go

```diff
@@ -39,6 +39,11 @@ type EthAPIBackend struct {
 	allowUnprotectedTxs bool
 }
 
+// SetExtRPCEnabled updates extRPCEnabled
+func (b *EthAPIBackend) SetExtRPCEnabled(v bool) {
```

Comment:
Setter is not needed, just assign it directly.

Reply:
I just wanted to highlight that this variable may be initialized after object creation